### PR TITLE
Drop Python 3.6 & 3.7 support

### PR DESCRIPTION
**Pull request by simonbohnen**
Should be merged after #46.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        PYTHON_VERSION: ["3.6", "3.7", "3.8", "3.9"]
+        PYTHON_VERSION: ["3.8", "3.9"]
```
Comment (ivergara) [inline .github/workflows/ci.yml:11]: Would add 3.10 by now.

Comment (simonbohnen) [reply .github/workflows/ci.yml:11]: Yep, that'll be done in #46!

Comment (ivergara) [reply .github/workflows/ci.yml:11]: Right! Sorry for forgetting that one!

**Review by xhochy — CHANGES_REQUESTED** [review]
Please add a `python_requires` to the `setup.py`, change the classifiers there and remove the `setuptools_scm` pin.

**Review by xhochy — APPROVED** [review]

**Pull request merged**